packages/react-charting/src/components/VerticalBarChart/tests/test-data.ts

```diff
@@ -0,0 +1,28 @@
+import { DefaultPalette } from '@fluentui/react/lib/Styling';
+
+export const chartPoints = [
```

Comment:
extracted generic fixture to separate module. without this change, BarChart test were re-generated within BarChartRTL test causing non deterministic output.